mayastor-test/e2e/common/util.go

```diff
@@ -90,6 +94,22 @@ func GetMSV(uuid string) *MayastorVolStatus {
 				}
 				Expect(msVol.Node != "")
 				break
+			case "replicas":
+				replicas := val.Interface().([]interface{})
+				for _, replica := range replicas {
+					replicaMap := reflect.ValueOf(replica)
+					if replicaMap.Kind() == reflect.Map {
+						for _, field := range replicaMap.MapKeys() {
+							switch field.Interface().(string) {
+							case "node":
+								value := replicaMap.MapIndex(field)
+								msVol.Replicas = append(msVol.Replicas, value.Interface().(string))
+								break
+							}
+						}
+					}
+				}
+				break
 			}
 		}
 	}
```

Comment:
The way I understand we count replicas is everything is replica so 1 replica is minimum (the one that nexus is running on and is attached as `bdev://`), isn't it? Shouldn't we test that we have at least one replica otherwise something is wrong? I.e. `Expect(len(msVol.Replicas)>0)`

Reply:
Yes, that sounds reasonable. It could also catch a problem if the MSV definition changes so that replicas are no longer found.